components/features/github/github-activity.tsx

```
@@ -0,0 +1,202 @@
+          aria-label="View William Callahan's GitHub profile"
+          title="View William Callahan's GitHub profile"
+        >
+          GitHub
+        </a>.
+      </p>
+      <div
+        onClick={navigateToGitHub}
+        className="border rounded-lg p-2 sm:p-4 bg-background overflow-hidden hover:border-blue-300 dark:hover:border-blue-700 transition-colors cursor-pointer"
+        aria-label="View William Callahan's GitHub contribution activity"
+        title="View William Callahan's GitHub profile"
+        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)